awx/main/models/inventory.py

```diff
@@ -914,23 +914,6 @@ class InventorySourceOptions(BaseModel):
 
     injectors = dict()
 
-    SOURCE_CHOICES = [
-        ('file', _('File, Directory or Script')),
```

Comment:
Do we have these text explanations anywhere? Or do we need to have them in the plugins interfaces?

Reply:
This item will be addressed later in another PR. Thank you for making sure this gets addressed!